src/components/WebexLocalMedia/WebexLocalMedia.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {WebexAvatar} from '../';
+import {useMeeting} from '../hooks';
+
+import './WebexLocalMedia.scss';
+
+/**
+ * Webex Local Media component displays the user's local video
+ * previous to joining a meeting from a given meetingID.
+ *
+ * @param {object} props
+ * @returns {object} JSX of the component
+ */
+export default function WebexLocalMedia({meetingID, personID}) {
+  const {localVideo} = useMeeting(meetingID);
+  const media =
+    localVideo && Object.entries(localVideo).length > 0 ? (
+      <video controls>
+        <source src={localVideo.src} />
```

Comment:
In the [WebRTC samples](https://github.com/webrtc/samples/blob/gh-pages/src/content/getusermedia/gum/js/main.js#L22), they set [`srcObject`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/srcObject) instead of `src`.

`srcObject` from the browser support perspective, mostly only works with `MediaStream` objects (like the ones the SDK will be sending), and `src` is meant to be used for URLS, files or blobs. We should follow their `Supporting fallback to the src property` sample code.

![Screen Shot 2019-11-20 at 11 45 33 AM](https://user-images.githubusercontent.com/6610308/69272328-543f0f80-0b8b-11ea-90cf-efae80321044.png)

Reply:
you want to embed the object this way instead of using `JSX` and attributes? the object can NOT be passed as an attribute.

Reply:
Right, that is how is handled in all examples of media I've seen

Reply:
we need to discuss about this then.

Reply:
this is why we are using `source` DOM